src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service.operation;
+
+public interface OperationHandler {
+    Integer calculateQuantity(Integer before, Integer after);
```

Comment:
let handler take transaction and change storage state accordingly
```suggestion
    void process(Transaction transaction);
```